lib/Console/Command/Parse.php

```diff
@@ -35,36 +43,21 @@ class Parse extends \Symfony\Component\Console\Command\Command
 
     protected function configure() {
         $this->setName('parse')
-             ->setDescription('Parse data and generate XML')
-        ;
+             ->setDescription('Parse data and generate XML');
     }
 
     protected function execute(InputInterface $input, OutputInterface $output) {
         $gl3Path = \Mesamatrix::$config->getValue('git', 'gl3', 'docs/GL3.txt');
-        $gitLog = new \Mesamatrix\Git\ProcessBuilder(array(
-            'log', '--pretty=format:%H|%at|%aN|%cN|%ct', '--reverse',
-            \Mesamatrix::$config->getValue('git', 'oldest_commit').'..', '--',
-            $gl3Path
-        ));
-        $gitLogProc = $gitLog->getProcess();
-        $this->getHelper('process')->mustRun($output, $gitLogProc);
 
-        $commitLines = explode(PHP_EOL, $gitLogProc->getOutput());
-        if (empty($commitLines)) {
+        $commits = $this->fetchCommits($gl3Path, $output);
+        if ($commits === NULL) {
```

Comment:
Either keep this block or keep the one in `fetchCommits()`, not both (you get two log entries otherwise)